src/llama-batch.cpp

```diff
@@ -370,34 +370,28 @@ static struct llama_batch_ext * llama_batch_ext_init_impl(int32_t n_tokens_alloc
     return batch;
 }
 
-struct llama_batch_ext * llama_batch_ext_init(int32_t n_tokens_alloc, int32_t n_seq_max) {
-    return llama_batch_ext_init_impl(n_tokens_alloc, 0, n_seq_max);
+struct llama_batch_ext * llama_batch_ext_init(struct llama_context * ctx) {
+    return llama_batch_ext_init_impl(llama_n_batch(ctx), 0, llama_n_seq_max(ctx));
```

Comment:
Yes, for now this is a good solution. Later, we will be resizing dynamically and not need the `llama_n_batch()`.